Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -431,6 +431,14 @@ def get_playbook_suggestion_by_issue(self, issue_id):
 
         return reply
 
+    def get_appsec_suggested_fix(self, issue_id: str) -> dict | None:
+        reply = self._http_request(
+            method="GET",
+            headers=self._headers,
+            full_url=f"/api/webapp/public_api/appsec/v1/issues/fix/{issue_id}/fix_suggestion",
```

Comment:
I think hezi used this prefix as a constant please look at the update scanner code.